components/PromptRemove.vue

```diff
@@ -2,23 +2,32 @@
 import { mapState, mapGetters } from 'vuex';
 import { get, isEmpty } from '@/utils/object';
 import Card from '@/components/Card';
+import Checkbox from '@/components/form/Checkbox';
 import { alternateLabel } from '@/utils/platform';
 import { uniq } from '@/utils/array';
 import AsyncButton from '@/components/AsyncButton';
+import { CATALOG as CATALOG_ANNOTATIONS } from '@/config/labels-annotations';
+import { CATALOG } from '@/config/types';
 
 export default {
-  components: { Card, AsyncButton },
+  components: {
+    Card, Checkbox, AsyncButton
+  },
   data() {
     const { resource } = this.$route.params;
 
     return {
-      hasCustomRemove: false,
-      randomPosition:  Math.random(),
-      confirmName:     '',
-      error:           '',
-      warning:         '',
-      preventDelete:   false,
-      removeComponent: this.$store.getters['type-map/importCustomPromptRemove'](resource)
+      hasCustomRemove:     false,
+      randomPosition:      Math.random(),
+      confirmName:         '',
+      error:               '',
+      warning:             '',
+      preventDelete:       false,
+      removeComponent:     this.$store.getters['type-map/importCustomPromptRemove'](resource),
+      deleteCrd:           false,
+      allInstalled:        null,
+      chartsToRemoveIsApp: false,
+      chartsDeleteCrd:     false
```

Comment:
This is set/reset but isn't used anywhere

Reply:
When I wrote this the first time I was using two different variables to achieve the same thing. So I removed `deleteCrd` and replaced where I was arguing against it with `chartsDeleteCrd`. Now if the checkbox is unchecked, `chartsDeleteCrd` will be false and the selected CRD will be removed from the `toRemove` array.